databases/redis-spotahome/terraform/gke-autopilot/versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 5.0"
```

Comment:
Note that Renovate will try to automatically upgrade these again as soon as this PR merges in.

Is there any reason latest TF provider is not compatible?

If yes and we can't go around it, you might have to either a) update the CI job such that it fails if a new PR upgrading to 5 occurs again or b) tweak the Renovate config to permanently set this version to 4

Reply:
Yep, when I use ~> 5.0 i get failed:
![Screenshot 2023-10-13 at 00 51 59](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/assets/52278182/05108986-4866-4c81-abd6-df4b1ef91a98)

Other people from the team also got the same issue, so i decided to downgrade the version (it used to work with ~> 4.0)

Reply:
Would you mind updating the databases/ CI with an additional job for this "redis-spotahome" directory, with steps for each TF to do `terraform init` + `terraform validate`?

That way when Renovate creates another PR to bump to v5 of the provider, we'll immediately see those checks fail and won't merge it accidentally

Reply:
Added CI steps